From d22f14236ca6217b10ce64e64bdf91038234cfb3 Mon Sep 17 00:00:00 2001
From: Pratham Deshmukh <p-deshmukh@ti.com>
Date: Wed, 4 Mar 2026 16:06:24 +0530
Subject: [PATCH 8/8] Add FP16 source path

Upstream-Status: Pending

Signed-off-by: Pratham Deshmukh <p-deshmukh@ti.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2df5622d47..a09bb27387 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -243,7 +243,7 @@ set_target_properties(
 )

 if(NOT (ACL_ARCH_ISA STREQUAL "armv7" OR ACL_ARCH_ISA STREQUAL "armv8-a"))
-  add_library(arm_compute_core_fp16 OBJECT )
+  add_library(arm_compute_core_fp16 OBJECT ${ARM_COMPUTE_CORE_FP16_SOURCES})
   set_target_properties(
     arm_compute_core_fp16
     PROPERTIES
--
2.34.1
